src/libraries/SqrtPriceMath.sol

```diff
@@ -169,6 +169,15 @@ library SqrtPriceMath {
         }
     }
 
+    /// @notice Equivalent to: `a > b ? a - b : b - a`
+    function absDiff(uint160 a, uint160 b) internal pure returns (uint256 res) {
+        assembly {
+            let diff := sub(a, b)
+            let mask := sar(255, diff)
```

Comment:
could you add a similar comment to absTick? just something like
mask is all 1s if b > a or all 0s when a >= b
b-a = -(a-b) = invert (a-b-1)